pom.xml

```diff
@@ -3997,6 +4018,12 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
```

Comment:
Seems to be only used in tests. The scope should be test, if needed here at all. I think the testsuite sets this up, and I don't see any other usage introduced by it here.